test/unit/auth/token-verifier.spec.ts

```diff
@@ -45,6 +46,12 @@ const expect = chai.expect;
 const ONE_HOUR_IN_SECONDS = 60 * 60;
 const idTokenPublicCertPath = '/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com';
 
+const AUTH_ERROR_CODE_CONFIG: ErrorCodeConfig = {
```

Comment:
Why not reuse the const defined in the implementation?

Reply:
Updated to reuse the ones from the implementation. Thanks!